lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void testCreateClassificationWithMasterCopy() throws Exception {
-    final long countClassificationsBefore =
-        CLASSIFICATION_SERVICE.createClassificationQuery().count();
+  void should_CreateMasterClassification_WhenCreatingClassificationWithDomain() throws Exception {
     Classification classification =
-        CLASSIFICATION_SERVICE.newClassification("Key1", "DOMAIN_A", "TASK");
-    classification.setIsValidInDomain(true);
-    classification.setServiceLevel("P1D");
-    classification = CLASSIFICATION_SERVICE.createClassification(classification);
-
-    // Check returning one is the "original"
-    Classification createdClassification =
-        CLASSIFICATION_SERVICE.getClassification(classification.getId());
-    assertThat(classification).isNotNull();
-    assertThat(classification.getCreated()).isNotNull();
-    assertThat(classification.getModified()).isNotNull();
-    assertThat(classification.getId()).isNotNull();
-    assertThat(classification.getIsValidInDomain()).isTrue();
-    assertThat(classification.getId()).startsWith(IdGenerator.ID_PREFIX_CLASSIFICATION);
-    assertThat(createdClassification.getDomain()).isEqualTo("DOMAIN_A");
-    assertThat(createdClassification.getKey()).isEqualTo("Key1");
-
-    // Check 2 new created
-    long amountOfClassificationsAfter = CLASSIFICATION_SERVICE.createClassificationQuery().count();
-    assertThat(amountOfClassificationsAfter).isEqualTo(countClassificationsBefore + 2);
-
-    // Check main
-    classification = CLASSIFICATION_SERVICE.getClassification(classification.getId());
-    assertThat(classification).isNotNull();
-    assertThat(classification.getCreated()).isNotNull();
-    assertThat(classification.getModified()).isNotNull();
-    assertThat(classification.getId()).isNotNull();
-    assertThat(classification.getIsValidInDomain()).isTrue();
-    assertThat(classification.getId()).startsWith(IdGenerator.ID_PREFIX_CLASSIFICATION);
+        classificationService.newClassification("Key1", "DOMAIN_A", "TASK");
 
-    // Check master-copy
-    classification = CLASSIFICATION_SERVICE.getClassification(classification.getKey(), "");
-    assertThat(classification).isNotNull();
-    assertThat(classification.getCreated()).isNotNull();
-    assertThat(classification.getModified()).isNotNull();
-    assertThat(classification.getId()).isNotNull();
-    assertThat(classification.getIsValidInDomain()).isFalse();
-    assertThat(classification.getId()).startsWith(IdGenerator.ID_PREFIX_CLASSIFICATION);
+    classificationService.createClassification(classification);
+
+    ClassificationImpl expectedMasterClassification =
+        (ClassificationImpl) classification.copy("Key1");
+    expectedMasterClassification.setDomain("");
+
+    List<ClassificationSummary> classifications =
+        classificationService.createClassificationQuery().keyIn("Key1").list();
+
+    assertThat(classifications)
```

Comment:
I think that it is important. The question I'm asking myself is if we should do this in a seperate test ...
We could do this by filtering the Query with idIn(...). This would be possible for other tests too. What do you think of this? We wouldn't need a seperate test and it would test it indirectly. In addition, we already have "should_SetDefaultValues_When_CreatingClassificationWithoutSpecificValues()" which tests if theres an id but of course only with default values ...